### PR TITLE
Added AR Marker sample

### PR DESCRIPTION
**Pull request by devdeeprc**

### CODE REVIEW

File: SampleScenarios/Scenarios/MarkerSample/Scripts/SampleMarker.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        {
+            if (m_marker != null && m_markerRenderer != null)
+            {
+                if (Time.realtimeSinceStartup - m_marker.lastSeenTime > 10)
+                {
+                    m_markerRenderer.material.color = Color.gray;
+                }
+                else
+                {
+                    m_markerRenderer.material.color = Color.red;
```
Comment (KBonds) [inline SampleScenarios/Scenarios/MarkerSample/Scripts/SampleMarker.cs:38]: Could we use a color other than red to indicate the "good" state?
At first, when I saw these alternate between red and grey, I assumed the red was for tracking problems

Comment (devdeeprc) [reply SampleScenarios/Scenarios/MarkerSample/Scripts/SampleMarker.cs]: Changed it to blue.

**Review by KBonds — APPROVED** [review]

File: SampleScenarios/Scenarios/MarkerSample/Scripts/MarkerSampleInstructions.cs [hunk 2]
```diff
@@ -14,7 +14,8 @@ private void Start()
             m_text = @"Sample: ARMarker
 
 - Use the 'Change Transform Mode' button to alter the center of the marker between origin and geometric center.
-- Use the 'Change Default Transform Mode' button to change the defaul transform mode of newly detected markers.";
+- Use the 'Change Default Transform Mode' button to change the default transform mode of newly detected markers.
+- A blue cube indicates a marker thats being tracked. A grey cube indicates a marker with limited tracking.";
```
Comment (KBonds) [inline SampleScenarios/Scenarios/MarkerSample/Scripts/MarkerSampleInstructions.cs:18]: nit: "thats being tracked" -> "that's being tracked"
Thanks for switching the color here too!

Comment (devdeeprc) [reply SampleScenarios/Scenarios/MarkerSample/Scripts/MarkerSampleInstructions.cs]: Addressed

**Pull request merged**